static/script-tests/tests/devices/mediaplayer/cehtmlcommontests.js

```diff
@@ -1369,6 +1369,46 @@ window.commonTests.mediaPlayer.cehtml.mixinTests = function (testCase, mediaPlay
         });
     };
 
+    mixins.testGetDurationReturnsInfinityWithALiveVideoStream = function(queue) {
```

Comment:
As returning infinity for a live stream is now the expecting behaviour across _all_ device types, would it make sense to put these tests in the common unit tests, instead of duplicating them for each modifier?